homeassistant/components/zwave_js/config_flow.py

```diff
@@ -149,7 +149,14 @@ def get_usb_ports() -> dict[str, str]:
             pid,
         )
         port_descriptions[dev_path] = human_name
-    return port_descriptions
+
+    # Sort the dictionary by description, putting "n/a" last
+    return dict(
+        sorted(
+            port_descriptions.items(),
+            key=lambda x: x[1].lower().startswith("n/a"),
```

Comment:
I guess the tuple was to sort 🔡  too. We could add that back.

Reply:
Merged before I saw this comment but that sort wasn't important imo